pkgs/applications/misc/pipr/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+, bubblewrap
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pipr";
+  version = "v0.0.12";
+
+  src = fetchFromGitHub {
```

Comment:
I think it would be better if we used the upstream, not the mirror, don't you? `fetchGit` + https://gitlab.com/Elkowar/pipr

Reply:
generally yes. In this case, I'm planning on moving my projects over to github completely in the near future, so i makes sense to directly use github. If using a mirror in general is an issue, I can use gitlab here of course

Reply:
Whatever works for you.